src/main.js

```diff
@@ -1,9 +1,13 @@
 import IntlPolyfill from "./core.js";
 
 // hack to export the polyfill as global Intl if needed
-if (typeof Intl !== undefined) {
-    Intl = IntlPolyfill;
-    IntlPolyfill.__applyLocaleSensitivePrototypes();
+if (typeof Intl !== 'undefined') {
+    try {
```

Comment:
Honesty I'm not sure about it too. It happened on the following CI test and I couldn't reproduce it on my local in spite of using same version of `Node.js`/`npm`, but I could see `Cannot assign to read only property`. That's why I added it.
https://travis-ci.org/andyearnshaw/Intl.js/jobs/126101600